tests/check_consistent.py

```diff
@@ -16,7 +20,64 @@
 ]
 
 
-def main():
+def assert_stubs_only(directory):
+    """Check that given directory contains only valid stub files."""
+    top = directory.split(os.sep)[-1]
+    assert top.isidentifier(), "Bad directory name: {}".format(top)
+    for _, dirs, files in os.walk(directory):
+        for file in files:
+            name, ext = os.path.splitext(file)
+            assert name.isidentifier(), "Files must be valid modules"
```

Comment:
Should add the name to the assertion failure to make the failure more debuggable. Same for a number of other checks below.

(Also, is there a reason not to use f-strings here?)